include/iotc.h

```diff
@@ -328,26 +269,10 @@ extern iotc_state_t iotc_publish(iotc_context_handle_t iotc_h,
                                  void* user_data);
 
 /**
- * @brief Publish binary data to Cloud IoT Core on a given topic.
- *
- * To publish a message, the device must already be connected to Cloud IoT Core.
- * The Device SDK calls <code>iotc_connect()</code> or
- * <code>iot_connect_to()</code> to connect to Cloud IoT Core.
- *
- * Unlike <code>iotc_publish()</code>, <code>iotc_publish_data()</code>
- * accepts a pointer and data length instead of a null-terminated string.
+ * @brief Publishes binary data to an MQTT topic.
  *
- * @param [in] data The message payload.
- * @param [in] data_len The size of the message, in bytes.
- *
- * @see iotc_create_context
- * @see iotc_publish
- * @see iotc_publish_data
- *
- * @retval IOTC_STATE_OK The publication request is correctly formatted.
- * @retval IOTC_OUT_OF_MEMORY The platform doesn't have enough memory
- *     to fulfull the request.
- * @retval IOTC_INTERNAL_ERROR An unrecoverable error occurred.
+ * @param [in] data A pointer to a buffer with the message payload.
+ * @param [in] data_len The size, in bytes, of the message.
```

Comment:
other parameters are same as for iotc_publish

Reply:
Done